### PR TITLE
	Uncommented lines in scripts that allowed them to run faster

### PR DESCRIPTION
**Pull request by achiu17**

### CODE REVIEW

**Comment by achiu17** [comment]
Uncommented lines in scripts that allowed them to run faster

File: experiments/selection-runner.sh [hunk 1]
```diff
@@ -23,14 +23,14 @@ while [ "$index" -lt "$count" ]; do
     cd ..
 
     echo 'Running prioritization for original order'
-    java -Xms1g -Xmx2g -cp ${newExperimentsCP[$index]} edu.washington.cs.dt.impact.Main.OneConfigurationRunner -technique prioritization -coverage statement -order original -origOrder ${newDirectories[$index]}/${experiments[$index]}-$k-order -testInputDir ${directories[$index]}/sootTestOutput -filesToDelete ${newDirectories[$index]}/${experiments[$index]}-env-files -project ${experiments[$index]} -testType $k -outputDir ./${seleDir} -timesToRun ${medianTimes} -getCoverage
+    java -Xms1g -Xmx2g -cp ${newExperimentsCP[$index]} edu.washington.cs.dt.impact.Main.OneConfigurationRunner -technique prioritization -coverage statement -order original -origOrder ${newDirectories[$index]}/${experiments[$index]}-$k-order -testInputDir ${directories[$index]}/sootTestOutput -filesToDelete ${newDirectories[$index]}/${experiments[$index]}-env-files -project ${experiments[$index]} -testType $k -outputDir ./${seleDir} -timesToRun ${medianTimes}
```
Comment (winglam) [inline experiments/selection-runner.sh:26]: You don't have to undo this line

File: experiments/prioritization-runner.sh [hunk 2]
```diff
@@ -15,14 +15,12 @@ while [ "$index" -lt "$count" ]; do
     echo 'Generating sootTestOutput'
   	java -cp ${sootCP[$index]} edu.washington.cs.dt.main.ImpactMain -inputTests ${experiments[$index]}-$k-order
 
-    java -cp ${experimentsCP[$index]} edu.washington.cs.dt.impact.Main.OneConfigurationRunner -technique prioritization -coverage statement -order original -origOrder ${experiments[$index]}-$k-order -testInputDir sootTestOutput -filesToDelete ${experiments[$index]}-env-files -project ${experiments[$index]} -testType $k -outputDir ../${priorDir} -timesToRun ${medianTimes} -getCoverage
-
```
Comment (winglam) [inline experiments/prioritization-runner.sh:19]: You don't have to undo this line.

File: experiments/parallelization-runner.sh [hunk 3]
```diff
@@ -13,9 +13,9 @@ while [ "$index" -lt "$count" ]; do
   	java -cp ${sootCP[$index]} edu.washington.cs.dt.main.ImpactMain -inputTests ${experiments[$index]}-$j-order
 
     echo 'Running prioritization for original order'
-    java -Xms1g -Xmx2g -cp ${experimentsCP[$index]} edu.washington.cs.dt.impact.Main.OneConfigurationRunner -technique prioritization -coverage statement -order original -origOrder ${experiments[$index]}-$j-order -testInputDir sootTestOutput -filesToDelete ${experiments[$index]}-env-files -project ${experiments[$index]} -testType $j -outputDir ../${paraDir} -timesToRun ${medianTimes} -getCoverage
+    java -Xms1g -Xmx2g -cp ${experimentsCP[$index]} edu.washington.cs.dt.impact.Main.OneConfigurationRunner -technique prioritization -coverage statement -order original -origOrder ${experiments[$index]}-$j-order -testInputDir sootTestOutput -filesToDelete ${experiments[$index]}-env-files -project ${experiments[$index]} -testType $j -outputDir ../${paraDir} -timesToRun ${medianTimes}
```
Comment (winglam) [inline experiments/parallelization-runner.sh:16]: You don't have to undo this line.

**Pull request merged**